backend/core/src/shared/email.service.ts

```diff
@@ -34,6 +37,43 @@ export class EmailService {
     await this.send(user.email, "Welcome to Bloom", this.template("register-email")({ user: user }))
   }
 
+  public async confirmation(user: User, listing: Listing, application: Application) {
+    let whatToExpectText
+    const listingUrl = `${process.env.APP_URL}/listing/${listing.id}`
+    const compiledTemplate = this.template("confirmation")
+
+    if (process.env.NODE_ENV == "production") {
+      Logger.log(
```

Comment:
This seems like it might be more of a `debug` level logging? Or maybe that's a larger conversation that needs to be had about how much request-level logging is happening...

Reply:
Maybe Logger.info? I'd be inclined to leave it on by default since it doesn't seem too noisy, but I agree we should get in the habit of being explicit about log levels.

Reply:
@bk3c I don't see `Logger.info` options are `'log' | 'error' | 'warn' | 'debug' | 'verbose'` I think thats sth you have added, Ben before to `welcome` action, so I just repeated the pattern. Not sure how error logging services work for NestJs. But I would say we don't want to have it here at all (if airbrake or sth can catch it) or have that bug tracking library here to catch it.